pkgs/by-name/hx/hxtools/package.nix

```
@@ -19,6 +20,7 @@ stdenv.mkDerivation (finalAttrs: {
   nativeBuildInputs = [
     pkg-config
     zstd
+    perl
   ];
 
   buildInputs = [
```

```suggestion
  ];

  buildInputs = [
    perl
```

`buildInputs` would be more correct, as we want `pkgsHostTarget.perl` here.